ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,56 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the antrea project. It deploys Kubernetes with the antrea image and tests wither the OVS have hardware offload capabilities or not.
```

Comment:
antrea -> Antrea

Reply:
wither -> whether
OVS have -> OVS has

Reply:
But is it to test whether OVS has offload capability, or test Antrea works well with OVS hardware offload?